core/chains/evm/logpoller/log_poller.go

```diff
@@ -714,6 +751,7 @@ func (lp *logPoller) LatestLogEventSigsAddrsWithConfs(fromBlock int64, eventSigs
 // response contains blocks in the same order as "numbers" in request parameters
 // the first context parameter takes precedence over contexts passed through qopts
 func (lp *logPoller) GetBlocks(ctx context.Context, numbers []uint64, qopts ...pg.QOpt) ([]LogPollerBlock, error) {
+	defer lp.postQuery(time.Now(), "GetBlocks")
```

Comment:
I wonder if we'll find ourselves wishing we knew how _many_ blocks were requested - otherwise this data seems too fuzzy to infer much from. Would it make sense to normalize this as a per-block metric?

Reply:
I forgot to close this PR. Yes, knowing number of blocks will be helpful and we already added relevant logs instead of metrics - https://github.com/smartcontractkit/chainlink/blob/99984afa591167bb2d250c61e00dcb6c25d74b1c/core/chains/evm/logpoller/log_poller.go#L797